src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -1949,7 +1949,7 @@ bool ApplicationManagerImpl::Stop() {
   application_list_update_timer_.Stop();
   try {
     SetUnregisterAllApplicationsReason(
-      mobile_api::AppInterfaceUnregisteredReason::IGNITION_OFF);
+        mobile_api::AppInterfaceUnregisteredReason::IGNITION_OFF);
```

Comment:
@KVGrygoriev make this change as additional commit

Reply:
@LuxoftAKutsan this changes will disappear when someone fix it in develop branch

Reply:
@KVGrygoriev commit will still contain this changes and cherry pick may trigger conflicts